### PR TITLE
Reorganize the deployment-tools.md file

### PR DESCRIPTION
**Pull request by nghi-ly**
## What are you changing in this pull request and why?

Audit the `deployment-tools.md` file and reorganize it.

h/t to [@willdavies0](https://github.com/willdavies0) for the Orchestra content! 

closes https://github.com/dbt-labs/docs-internal/issues/603

## Checklist
- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [x] Needs review from DX eng

### CODE REVIEW

File: website/docs/docs/deploy/deployment-tools.md [hunk 1]
```diff
@@ -53,10 +57,33 @@ To use the dbt API to trigger a job in dbt Cloud through ADF:
...
+
+## Orchestra
+
+If your organization uses [Orchestra](https://getorchestra.io), you can trigger dbt jobs using the dbt Cloud API. Create an API token from your dbt Cloud account and use this to authenticate Orchestra in the [Orchestra Portal](https://app.getorchestra.io). For details, refer to the Orchestra docs on [dbt Cloud](https://orchestra-1.gitbook.io/orchestra-portal/integrations/transformation/dbt-cloud).
+
+The following are examples of
```
Comment (joellabes) [inline website/docs/docs/deploy/deployment-tools.md]: This trails off

File: website/docs/docs/deploy/deployment-tools.md [hunk 2]
```diff
@@ -53,10 +57,33 @@ To use the dbt API to trigger a job in dbt Cloud through ADF:
 5. Trigger the pipeline in ADF to start the dbt Cloud job and monitor the status of the dbt Cloud job in ADF.
 6. In dbt Cloud, you can check the status of the job and how it was triggered in dbt Cloud.
 
+## Cron
+
+Cron is a decent way to schedule bash commands. However, while it may seem like an easy route to schedule a job, writing code to take care of all of the additional features associated with a production deployment often makes this route more complex compared to other options listed here.
+
+## Dagster
+
+If your organization uses [Dagster](https://dagster.io/), you can use the [dagster_dbt](https://docs.dagster.io/_apidocs/libraries/dagster-dbt) library to integrate dbt commands into your pipelines. This library supports the execution of dbt through dbt Cloud, dbt Core, and the dbt RPC server. Running dbt from Dagster automatically aggregates metadata about your dbt runs. Refer to the [example pipeline](https://dagster.io/blog/dagster-dbt) for details.
```
Comment (joellabes) [inline website/docs/docs/deploy/deployment-tools.md]: The rpc server is deprecated

```suggestion
If your organization uses [Dagster](https://dagster.io/), you can use the [dagster_dbt](https://docs.dagster.io/_apidocs/libraries/dagster-dbt) library to integrate dbt commands into your pipelines. This library supports the execution of dbt through dbt Cloud or dbt Core. Running dbt from Dagster automatically aggregates metadata about your dbt runs. Refer to the [example pipeline](https://dagster.io/blog/dagster-dbt) for details.
```

**Review by joellabes — CHANGES_REQUESTED** [review]
Two comments otherwise lgtm!

**Review by joellabes — APPROVED** [review]

**Review by runleonarun — APPROVED** [review]
This looks good!

**Pull request merged**